scripts/circle-release-next-packages.sh

```diff
@@ -64,23 +64,7 @@ else
 
   for PACKAGE in "${PACKAGES[@]}"
   do
-    start=$(date +%s%N)
-    yarn workspace @grafana/"${PACKAGE}" run build
-    runtime=$((($(date +%s%N) - start)/1000000))
-    if [ "${DRONE_BRANCH}" == "master" ]; then
-      exit_if_fail ./scripts/ci-metrics-publisher.sh "grafana.ci-buildtimes.${DRONE_STEP_NAME}.$PACKAGE=$runtime"
```

Comment:
In the new version of `grabpl` do we publish the same metrics if the build fails on master?

Reply:
@mckn Doesn't this logic publish build times per package? I don't really see how eventual failure matters.

That said, it isn't currently implemented in the build pipeline tool. I have a task to port build metrics to it.

Reply:
hmm, I agree. @dprokop do you remember why we shipped these metrics?